packages/plugin-ext/src/main/browser/plugin-frontend-contribution.ts

```diff
@@ -20,6 +20,10 @@ import { HostedPluginManagerClient, HostedPluginCommands } from '../../hosted/br
 import { PluginExtDeployCommandService } from './plugin-ext-deploy-command';
 import { OpenUriCommandHandler } from './commands';
 import URI from '@theia/core/lib/common/uri';
+import { ApplicationShell, NavigatableWidget, OpenerService, open, Saveable } from '@theia/core/lib/browser';
```

Comment:
don't forget to revert this file, i meant to cherry pick only to run tests locally